docs/source/markdown/podman-create.1.md

```diff
@@ -22,6 +22,28 @@ Default settings for flags are defined in `containers.conf`. Most settings for
 remote connections use the server's containers.conf, except when documented in
 man pages.
 
+## IMAGE
+
+ The `image` uses a "transport":"details" format.
+
+ Multiple transports are supported when run locally.  Only the `docker` (container registry)
+ transport is allowed for remote access.
+
+  **dir:**_path_
+  An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
```

Comment:
How do I make a directory like this? Providing a command would greatly help understanding.

Reply:
podman push DIR 
Would create this or podman pull DIR.